std/conv.d

```diff
@@ -2870,6 +2907,15 @@ Target parse(Target, Source)(ref Source p)
     return (sign) ? -ldval : ldval;
 }
 
+///
+@safe unittest
+{
+    import std.math : approxEqual;
+    auto str = "123.456";
+
+    assert(parse!double(str).approxEqual(123.456));
```

Comment:
I understand why `approxEqual` was chosen here, but surely parsing `"123.456"` should yield the *exact* result of the compiler parsing a double literal `123.456`? (According to [the spec](https://dlang.org/spec/lex.html#FloatLiteral) a float literal without suffix is of type `double`.) Otherwise something is broken with `parse`.